torch/onnx/_internal/exporter/_analysis.py

```diff
@@ -162,7 +162,17 @@ def _get_io_specs(exported_program: torch.export.ExportedProgram) -> tuple[dict,
     inputs: dict[str, torch._export.serde.schema.TensorMeta] = {}
     outputs: dict[str, torch._export.serde.schema.TensorMeta] = {}
     for spec in user_inputs:
-        if isinstance(spec.arg, graph_signature.ConstantArgument):
+        # If dynamic is set to a constant input, it becomes a
+        # symbolic argument, which is not a tensor.
+        if isinstance(
```

Comment:
Should we include them?

Reply:
They don't have tensor_meta though. Is there other info we might want from them?

Reply:
PTAL

Reply:
I think this is only for printing iirc? If so it's fine as long as we are able to display them in the report.